src/commands/init/presets.rs

```diff
@@ -0,0 +1,197 @@
+//! Preset configurations for the `init` command.
+//!
+//! This module provides predefined formatter configurations that can be used to quickly
+//! set up `mago.toml` with common coding standards. Each preset is a complete
+//! `[formatter]` TOML section.
+
+use strum_macros::EnumString;
+
+/// Enum representing the available formatter presets.
+#[derive(Clone, Debug, EnumString)]
+#[strum(serialize_all = "kebab-case")]
+pub enum FormatterPreset {
+    Laravel,
+    Psr12,
+    Default,
+}
+
+/// Returns the TOML configuration string for the given formatter preset.
+///
+/// # Arguments
+///
+/// * `preset` - The `FormatterPreset` to get the configuration for.
+///
+/// # Returns
+///
+/// A static string slice containing the TOML configuration for the formatter.
+pub fn get_preset_config(preset: FormatterPreset) -> &'static str {
+    match preset {
+        FormatterPreset::Laravel => LARAVEL_PRESET,
+        FormatterPreset::Psr12 => PSR12_PRESET,
+        FormatterPreset::Default => DEFAULT_PRESET,
+    }
+}
+
+/// The default formatter preset, based on PER-CS.
+const DEFAULT_PRESET: &str = r#"[formatter]
```

Comment:
I'm not sure having those constants as `&str` is a good idea, we might miss updating them if configurations change in the future..

I think its better to have them as an instant of `FormatterSettings`, and encode it when writing to configurable file.